README.md

```diff
@@ -74,10 +74,32 @@ e.g. the indication of a lifecycle state of a value by “as-specified” or “
 e.g. the indication of multiplicity of allowed components for modelling by “greater than 0” and others.
 (Answered: 2020-08-13)
 
-**Where are examples of AAS available?**
+**Where are examples of AAS available?** <!-- ID: 16 -->
 
 Examples of several suppliers are found at: http://admin-shell-io.com/samples/ 
 (Answered: 2020-08-13)
 
+**What are best practices for creating custom IRI identifiers for generic concepts.** <a name="id18"></a><!-- ID: 18 -->
 
+What are best practices for creating custom IRI identifiers for generic concepts.
+We advise using “https://admin-shell.io/” prefix for those identifiers (see guides via https://github.com/admin-shell-io/id/) for generic concepts. Note that we advise to use “https:” protocol and avoid adding “www.” subdomain into admin-shell.io naming scheme.
+Furthermore, in practical implementations we advise to “filter” out protocol and schema for comparing IRI identifiers. For example, following IDs should be considered equal:
+-	https://admin-shell.io/some_id_example
+-	http://admin-shell.io/some_id_example
+-	ftp://admin-shell.io/some_id_example
```

Comment:
ftp really equal to https etc.?

Reply:
Idea was to ignore the protocol-specific schema to make implementation practical

Reply:
2020-08-24 Meeting Exemplarische Modellierungen: no change needed, this is what should be done when comparing ids.